src/store/cache/analytics/hooks.ts

```diff
@@ -289,132 +255,86 @@ export function fetchTotalStaked(
-}
-
-const getTrailingAPILegacy = (
-  endpoint: string,
-  startTime: number
-) => async () => {
-  const url = `${endpoint}/v1/metrics/routes?bucket_size=century&start_time=${startTime}`
-  const json = await fetchWithTimeout(url)
+const getTrailingAPI = async (nodes: DiscoveryProvider[]) => {
+  const json = await Promise.any(
+    nodes.map(node => {
+      const url = `${node.endpoint}/v1/metrics/aggregates/routes/trailing/month`
+      return fetchWithTimeout(url)
+    })
+  )
   return {
-    count: json?.data?.[0]?.count ?? 0,
-    unique_count: json?.data?.[0]?.unique_count ?? 0
+    total_count: json?.data?.total_count ?? 0,
+    unique_count: json?.data?.unique_count ?? 0,
+    summed_unique_count: json?.data?.summed_unique_count ?? 0
   } as CountRecord
 }
 
 export function fetchTrailingApiCalls(
   bucket: Bucket,
   nodes: DiscoveryProvider[]
 ): ThunkAction<void, AppState, Audius, Action<string>> {
-  return async (dispatch, getState, aud) => {
-    const startTime = getStartTime(bucket)
+  return async dispatch => {
     let error = false
-    const datasets = (
-      await Promise.all(
-        nodes.map(async node => {
-          try {
-            const res = await performWithFallback(
-              getTrailingAPI(node.endpoint),
-              getTrailingAPILegacy(node.endpoint, startTime)
-            )
-            return res
-          } catch (e) {
-            console.error(e)
-            error = true
-            return {}
-          }
-        })
-      )
-    ).filter(Boolean)
+    let metric = {}
+    try {
+      metric = await getTrailingAPI(nodes)
+    } catch (e) {
+      console.error(e)
+      error = true
+    }
     if (error) {
       dispatch(setTrailingApiCalls({ metric: MetricError.ERROR, bucket }))
       return
     }
-    const metric = joinCountDatasets(datasets)
-
     dispatch(setTrailingApiCalls({ metric, bucket }))
   }
 }
 
-const getTrailingTopApps = (
-  endpoint: string,
+const getTrailingTopApps = async (
+  nodes: DiscoveryProvider[],
   bucket: Bucket,
   limit: number
-) => async () => {
-  const bucketPaths: { [bucket: string]: string | undefined } = {
-    [Bucket.WEEK]: 'week',
-    [Bucket.MONTH]: 'month',
-    [Bucket.ALL_TIME]: 'all_time'
-  }
-  const bucketPath = bucketPaths[bucket]
-  if (!bucketPath) throw new Error('Invalid bucket')
-  const url = `${endpoint}/v1/metrics/app_name/trailing/${bucketPath}?limit=${limit}`
-  const json = await fetchWithTimeout(url)
-  if (!json.data) return {}
-  return json
-}
-
-const getTopAppsLegacy = (
-  endpoint: string,
-  startTime: number,
-  limit: number
-) => async () => {
-  const url = `${endpoint}/v1/metrics/app_name?start_time=${startTime}&limit=${limit}&include_unknown=true`
-  const json = await fetchWithTimeout(url)
-  if (!json.data) return {}
-  return json
+) => {
+  const json = await Promise.any(
+    nodes.map(node => {
+      const url = `${node.endpoint}/v1/metrics/aggregates/apps/${bucket}?limit=${limit}`
```

Comment:
Just realized a slight subtle difference here -- we currently aggregate all calls that don't have an explicit app set to Audius Apps + Unknown. We should try and replicate that behavior here if possible. Might mean subtracting from total calls b/c we don't capture things this way anymore